services/wiki/wiki_path.go

```diff
@@ -141,8 +145,7 @@ func WebPathFromRequest(s string) WebPath {
 }
 
 func UserTitleToWebPath(base, title string) WebPath {
-	// TODO: ctx.Params does un-escaping, so the URL "/wiki/abc%2Fdef" becomes "wiki path = `abc/def`", which is incorrect.
-	// And the old wiki code's behavior is always using %2F, instead of subdirectory.
+	// TODO: the old wiki code's behavior is always using %2F, instead of subdirectory.
```

Comment:
What exactly would need to be done to resolve this TODO?
Please describe it directly in the TODO.

Reply:
And there is also a comment below:

`So we do not add the support for writing slashes in title at the moment.`

It's not resolved at the moment, unless enough people would like to implement the "sub-directory" support.